18F/C2
# [Delivers #95549162] delegates redirected to approval after login

Story: https://www.pivotaltracker.com/story/show/95549162

This changes the ReturnTo helper to use the session to temporarily store the original request path so that the post-login callback can redirect after successful authentication.

QA: You can test this feature:

* add yourself (Y) as a delegate to an existing User (X)
* create a new proposal listing X as the approver
* using the approval email sent to X, copy the Approve button link (should contain the **cch** token param)
* using a browser *not logged in to MyUSA or the app*, paste the link. 
* login as Y (the delegate)
* on redirect back to the app, your browser should redirect and result in the proposal being approved.
